forc-plugins/forc-publish/src/forc_pub_client.rs

```diff
@@ -50,16 +52,49 @@ impl ForcPubClient {
             .header("Content-Type", "application/gzip")
             .body(file_bytes)
             .send()
-            .await?;
-
-        let status = response.status();
+            .await;
 
-        if status.is_success() {
-            // Extract `upload_id` from the response if available
-            let upload_response: UploadResponse = response.json().await?;
-            Ok(upload_response.upload_id)
+        if let Ok(response) = response {
+            let mut stream = response.bytes_stream();
+
+            // TODO: Close stream
```

Comment:
```suggestion
            // TODO: Close stream
```

Is this something needed, or left in while development?